src/app/filecoin-plus/allocators/data/allocatorsTableColumnsData.tsx

```diff
@@ -32,17 +39,27 @@ export const allocatorsTableColumnsData = [
     filterFn: 'includesString',
   }),
   columnHelper.accessor('application.required_replicas', {
-    header: 'Required Replicas',
+    header: () => (
+      <TableHeaderWithTooltip
+        header="Required Replicas"
+        description="The minimum number of data copies that must be stored."
+      />
+    ),
     cell: (info) => info.getValue(),
     meta: {
-      headerCellStyle: 'w-44',
+      headerCellStyle: 'w-52',
     },
   }),
   columnHelper.accessor('application.required_sps', {
-    header: 'Required SPs',
+    header: () => (
+      <TableHeaderWithTooltip
+        header="Required SPs"
+        description="The minimum number of distinct storage providers needed for data storage."
+      />
+    ),
```

Comment:
I remember Stef recently pointing out in one of the Dogfooding sessions to write SPs out in full - We can check this with him if that's the case here as well. Either way, this does happen when the label is long. 

![CleanShot 2024-11-04 at 09 44 12@2x](https://github.com/user-attachments/assets/5e9b7b1a-b206-4d2c-b5d0-4192a0daad22)

Reply:
Remove the fixed layout on this table.
SP -> Storage Providers

Reply:
✅